docs/pages/vercel.page.mdx

```diff
@@ -2,24 +2,20 @@ import { RepoLink } from 'libframe-docs/components/RepoLink'
 
 To deploy a `vite-plugin-ssr` app to Vercel:
  1. Load `dist/server/importBuild.js` in your Vercel serverless function.
-    Example: <RepoLink path='/examples/vercel/api/render.js' />.
+    Example: <RepoLink path='/examples/vercel/vercel/render.js' />.
     > Behind the scenes, Vercel bundles your serverless function and therefore needs to know its entire dependency tree;
     > [`dist/server/importBuild.js` enables](/importBuild.js) Vercel to discover the entire dependency tree of your serverless function.
 
- 2. Define your build script in `package.json#scripts['vercel-build']` instead of `package.json#scripts.build`.
+ 2. Define deploy.sh script in `package.json#scripts['vercel-build']` along with `package.json#scripts.build`.
     Example: <RepoLink path='/examples/vercel/package.json' />.
     > Vercel discards all files generated by `package.json#scritps.build`. That's why we need to use `package.json#scripts['vercel-build']` instead. See [Vercel Docs - Custom Build Step for Node.js](https://vercel.com/docs/runtimes#advanced-usage/advanced-node-js-usage/custom-build-step-for-node-js).
 
- 3. Define a `package.json#scripts.build` that does nothing.
-    Example: <RepoLink path='/examples/vercel/package.json' />.
-    > We need to define a `pacakge.json#scripts.build`, otherwise Vercel ignores the generated static files living at `dist/client/`.
-
- 4. In Vercel's web UI, set the `OUTPUT DIRECTORY` config of your Vercel app to `dist/client` (and make sure to sure enable `OVERRIDE`).
+ 3. In Vercel's web UI, set the env variable `ENABLE_FILE_SYSTEM_API` of your Vercel app to `1`.
 
 Example:
  - <RepoLink path='/examples/vercel/' />
 
 More infos:
  - Insightful conversation: [github.com/vercel/vercel/discussions/6393](https://github.com/vercel/vercel/discussions/6393#discussioncomment-1008607).
  - `vercel.json`'s config `includeFiles` doesn't work for files that are not committed to your Git repository: [github.com/vercel/vercel/issues/6478](https://github.com/vercel/vercel/issues/6478).
-
+ - You can use `vercel.json` to handle redirects. Example: <RepoLink path='/examples/vercel/vercel.json' />
```

Comment:
Not sure if should add this note. I mean how Vercel supports redirects has not much to do with vite-plugin-ssr.